### PR TITLE
Update issue template to use discussions for questions

### PR DESCRIPTION
**Pull request by edenlightning**

### CODE REVIEW

**Review by Borda — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

**Comment by rohitgr7** [comment]
https://github.com/PyTorchLightning/pytorch-lightning/blob/master/notebooks/04-transformers-text-classification.ipynb
https://github.com/PyTorchLightning/pytorch-lightning/blob/master/notebooks/06-mnist-tpu-training.ipynb
notebooks too pls.

File: README.md [hunk 1]
```diff
@@ -391,10 +391,8 @@ Lightning is also part of the [PyTorch ecosystem](https://pytorch.org/ecosystem/
 ### Asking for help
 If you have any questions please:
 1. [Read the docs](https://pytorch-lightning.rtfd.io/en/latest).
-2. [Search through the Discussions](https://github.com/PyTorchLightning/pytorch-lightning/discussions).
-3. [Look it up in our forum (or add a new question)](https://forums.pytorchlightning.ai)
-4. [Join our slack](https://join.slack.com/t/pytorch-lightning/shared_invite/zt-f6bl2l0l-JYMK3tbAgAmGRrlNr00f1A).
-
+2. [Search through existing Discussions](https://github.com/PyTorchLightning/pytorch-lightning/discussions), [or add a new question](https://github.com/PyTorchLightning/pytorch-lightning/discussions/new?category=api-help-module-trainer-callbacks-datamodules)
```
Comment (rohitgr7) [inline README.md:394]:
```suggestion
2. [Search through existing Discussions](https://github.com/PyTorchLightning/pytorch-lightning/discussions), or [add a new question](https://github.com/PyTorchLightning/pytorch-lightning/discussions/new)
```
one can select the category

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Ask a Question
+    url: https://github.com/PyTorchLightning/pytorch-lightning/discussions/new?category=api-help-module-trainer-callbacks-datamodules
```
Comment (rohitgr7) [inline .github/ISSUE_TEMPLATE/config.yml:4]:
```suggestion
    url: https://github.com/PyTorchLightning/pytorch-lightning/discussions/new
```

**Comment by s-rog** [comment]
The slack url for the issue template and readme are different, should they be the same?

**Review by SkafteNicki — APPROVED** [review]

**Pull request merged**